server/assetstore.py

```diff
@@ -199,6 +199,8 @@ def downloadFile(self, file, offset=0, headers=True, endByte=None,
                     'The extraParameters field must either be a dictionary, a '
                     'JSON-encoded dictionary, or a url query-encoded string.')
             params.update(extraParameters)
+            if params.get('limit', 'notpresent') is None:
```

Comment:
I just want to make sure: this line draws a distinction between `'limit' not in params` (which will make this evaluate to `'notpresent'`) and `'limit' in params and params['limit'] == None` (which would make this evaluate to `None`). Is that right?